src/Engines/Nunjucks.js

```diff
@@ -166,6 +238,8 @@ class Nunjucks extends TemplateEngine {
       };
     }
 
+    // Invalidate the caches.
```

Comment:
Superfluous? The function name is clear enough.

Reply:
Fixed in updated patch.